test/data/brave_scheme_load.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<script>
+  const url = "brave://settings"
+  function openBraveSettings() {
+    window.open(url);
+  }
+
+  function replaceToBraveSettings() {
+    window.location.replace(url);
```

Comment:
this is also currently blocked for other reasons; `w = window.open('https://brave.com'); setTimeout(()=>{w.location.replace('brave://settings')},1000)` works though

Reply:
`NotAllowedToIndirectReplaceTest` case is added for async replace.
`window.location.replace(url)` also has same meaning as above.